content/posts/2025-10-02-WildFly38Beta-released.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "New WildFly 38 Beta release"
+aliases: [/news/2025/10/02/WildFly38Beta-released/]
+date:   2025-10-02
+tags:   announcement release
+author: darranl
+description: WildFly 38 Beta is now available for download.
+---
+
+We're excited to announce the release of *WildFly 38 Beta 1*! This version is contains updates, new features, and bug fixes, paving the way for the upcoming final release. This beta version is a crucial step in our development cycle, allowing the community to test and provide feedback on the latest changes.
+
+
+== What's New and Noteworthy?
+
+This release focuses heavily on *component upgrades*, *MicroProfile 7.1 support*, and enhancements to the *WildFly Preview* distribution. Here's a quick rundown of the highlights:
+
+* **Java EE 11 Integration:** A key new feature is the implementation of *Jakarta EE 11 JPA and CDI integration* in WildFly Preview, bringing us closer to full support for the latest Jakarta EE specifications.
+* **MicroProfile 7.1:** WildFly 38 Beta 1 supports the *MicroProfile 7.1* specification, including an update to the latest versions of SmallRye Reactive Messaging and Config. This upgrade ensures that developers can leverage the newest features and improvements in the MicroProfile ecosystem.
```

Comment:
s/SmallRye Reactive Messaging and Config/MicroProfile OpenAPI and MicroProfile Telemetry/g

I don't know if we also upgraded our RM and Config impls, but even if we did it's not the key point.

Also, perhaps

s/specification/specifications

I don't really think of MP platform as a specification the way Jakarta EE is. I don't even know if it has a spec document. I think of it as a collection of specifications.